ext/standard/tests/array/gh14775.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+GH-14775: Range negative step overflow
+--FILE--
+<?php
+$var = -PHP_INT_MAX - 1;
```

Comment:
why not PHP_INT_MIN?

Reply:
why not ? it s just a test case, relax :)

Reply:
IMO PHP_INT_MIN is much readable, isn't it?

Reply:
I agree this is way less readable.